src/EditorFeatures/Core/Shared/SuggestionSupport/DefaultDocumentSupportsSuggestionService.cs

```diff
@@ -22,5 +22,10 @@ public bool SupportsRename(Document document)
         {
             return true;
         }
+
+        public bool SupportsNavigationToAnyPosition(Document document)
```

Comment:
:+1: to @CyrusNajmabadi's comment. Maybe "DocumentSupportsDumpingGroundOfRandomBooleansService"? That seems to be what is is now.